File: libs/Result.rb
class Result
	def Result.reader
		Dir.chdir("Results")
		savefilename = $savefilename
		read = File.open("#{savefilename}.xrsbxl2x", "r")
<<<<<<< HEAD
		puts read.readlines
=======
		puts read.readline
		puts read.readline
		puts read.readline
		puts read.readline
		puts read.readline
>>>>>>> 3f177042cedc2a5d5dcc0a6704d036d0777d3606
		gets
	end
end

File: java/jabber/src/Jabber/src/jabber/core/Result.java
package jabber.core;

import fileClass.ReadFile;
import java.io.IOException;
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

/**
 *
 * @author onigiri
 */
public class Result {
    public String saveFileName;
    
    public Result(String savefilename) {
        saveFileName = savefilename;
    }
    
    public void readResults() {
        String saveFile = saveFileName + ".xrsbxl2x";
        ReadFile read = new ReadFile("./Results/" + saveFile);
        List<String> readLines = new ArrayList<>();
        try {
            Collections.addAll(readLines, read.OpenFile());
        } catch (IOException ex) {
            System.out.println(ex.getMessage());
            System.exit(-1);
        }
        String[] lines = readLines.toArray(new String[readLines.size()]);
        for (int i = 0; i < 5; i++) {
            System.out.println(lines[i]);
        }
    }
}
